src/main/java/com/example/CustomerService/entity/CustomerCredentials.java

```diff
@@ -0,0 +1,28 @@
+package com.example.CustomerService.entity;
+
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.validator.constraints.UniqueElements;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+public class CustomerCredentials {
```

Comment:
Make use of @JoinColumn and @OneToMany annotations by adding customer details entity for the users which are getting registered.